3rdparty/preact/signals/index.ts

```diff
@@ -420,3 +418,26 @@ export function update<T extends SignalOrReactive>(
 	}
 }
 */
+
+const enum OptionsTypes {
```

Comment:
FWIW, I haven't tried this yet, but it might be possible to reimplement this without an enum using a union of string literals, such as:

```
type OptionTypes = "_hook" | "_diff" | ...
```

Reply:
Noted =)